problem_builder/tests/integration/test_questionnaire.py

```diff
@@ -115,17 +112,9 @@ def submit_answer_and_assert_messages(mcq1_answer, mcq2_answer, item_feedback1,
             mcq1_tips = mcq1.find_element_by_css_selector(".choice-tips .tip p")
             mcq2_tips = mcq2.find_element_by_css_selector(".choice-tips .tip p")
 
-            self.assertTrue(mcq1_feedback.is_displayed())
```

Comment:
The test should be checking that if you choose an option with a tip, then `mcq1_tips.text` contains that tip. But if you choose an option without a tip, the test should make sure that `mcq1_tips.text` contains the general feedback. So your change to this method is fine, but we need to add some instances of

```python
submit_answer_and_assert_messages(..., "Feedback message 2")
```

to confirm the question-level feedback is working properly.